pkg/combustion/registry.go

```diff
@@ -0,0 +1,129 @@
+package combustion
+
+import (
+	_ "embed"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/suse-edge/edge-image-builder/pkg/fileio"
+	"github.com/suse-edge/edge-image-builder/pkg/image"
+	"github.com/suse-edge/edge-image-builder/pkg/log"
+	"github.com/suse-edge/edge-image-builder/pkg/template"
+)
+
+const (
+	haulerManifestYamlName = "hauler-manifest.yaml"
+	registryScriptName     = "13-embedded-registry.sh"
+	haulerTarName          = "haul.tar.zst"
```

Comment:
Can we rename this to `container-registry.tar.zst` or something similar instead?